IdelPog/Assets/Scripts/IdelPog/Structures/CurrencyTrade.cs

```diff
@@ -0,0 +1,38 @@
+﻿using IdelPog.Structures.Enums;
+
+namespace IdelPog.Structures
+{
+    /// <summary>
+    /// This structure is used to update a Currency model. 
+    /// </summary>
+    /// <remarks>
+    /// Uses the internal <see cref="CurrencyType"/> to dictate what Currency to update, this update is decided by the <see cref="ActionType"/>
+    /// </remarks>
+    /// <seealso cref="Amount"/>
+    /// <seealso cref="Currency"/>
+    /// <seealso cref="Action"/>
+    public struct CurrencyTrade
```

Comment:
I think all the properties in this struct should be just readonly, or just make it a get only prop.

Reply:
All props are now Get only.